README.md

```diff
@@ -56,6 +57,69 @@ Add new records to your mongodb instance by HTTP PUT using the following route:
 * `http://localhost:3000/scicat/Publication`
 
 
+## ENVIRONMENT Variables
+
+*System variables**
+ Key | Description | Default
+ --------:|-------------| --------
+DAPP_ID | No Idea | oai-pmh-service
+CONNECTOR | (don't change) | mongodb
+ADMIN_USER_EMAIL | E-Mail address of the admin user | none
+LOG_LEVEL | default/error/warning | none  
+
+**Mongo DB variables**
+ Key | Description | Default
+ --------:|-------------| --------
+CONNECTOR | (don't change) | mongodb
+DB_HOST | Database Hostname | none
+DB_PORT | Database Port | none
+DB_USER | Database Username | none
+DB_PASS | Database Password | none
+DB_URL |  [&lt;user&gt;:&lt;password&gt;@]&lt;host&gt;:&lt;port&gt;/&lt;dbName&gt;| none
+DATABASE | Publication Database | dacat-next
```

Comment:
maybe change to new scicat namings in a follow-up

Reply:
Ok will do later.